python/tls/setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup
+import os
+
+setup(name='fybrik_python_tls',
+      version=os.environ.get('VERSION', '0.1.0'),
```

Comment:
if you need this env variable,  so maybe "Python-TLS-Version" or something similar ? We can face with a conflict.

Reply:
Changed Thanks